src/types/api.ts

```diff
@@ -222,6 +228,8 @@ export type APIAnnotationData = {
    * current assignment, course etc. to annotations.
    */
   metadata?: object;
+
+  mentions: Mention[];
```

Comment:
Can you add a brief JSDoc comment to explain what this field is for. That seems obvious to us right now, but this might not be the case for a new contributor.